### PR TITLE
Camunda EE 7.13.1 Support

### PR DESCRIPTION
**Pull request by dbudke**
Support for Camunda EE 7.13.0
Added documentation for H2 support.
Removed undertow dependency since Tomcat is used
Added support for process testing using plain spring container and spring boot
Added support for process test coverage report for non spring boot tests
Added sample-process.bpmn illustrating process scenario testing
Extended readme.md

### CODE REVIEW

File: src/test/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/ProcessScenarioTest.java [hunk 1]
```diff
@@ -54,9 +55,7 @@ public void testHappyPath() {
     // when(myProcess.waitsAtReceiveTask(anyString())).thenReturn((messageSubscription) -> {
     //  messageSubscription.receive();
     // });
```
Comment (hallerch) [inline src/test/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/ProcessScenarioTest.java]: Is the comment needed? If the comment is needed than this code would most likely give a lambda warnings like on the lines 57-59 that you already changed to line 58.

**Review by hallerch — COMMENTED** [review]
Build was successful.
2 of 2 tests passed.
Code looks good for me.
One comment on ProcessScenarioTest.java lines 55 to 57

File: src/test/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/ProcessScenarioTest.java (hunk 1 above)
Comment (hallerch) [reply src/test/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/ProcessScenarioTest.java]: //when(myProcess.waitsAtReceiveTask(anyString())).thenReturn(EventSubscriptionDelegate::receive);

Comment (dbudke) [reply src/test/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/ProcessScenarioTest.java]: Updated the test case and added more examples including process test coverage report in target folder

Comment (hallerch) [reply src/test/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/ProcessScenarioTest.java]: When I run the tests for the first time then one test fails (ProcessIntegrationTest.java -> testHapyPath()).
When I run the tests a second time then 4 of 5 tests fails.  
(ProcessIntegrationTest.java -> testHapyPath() and ProcessSenarioTest.java -> testHappyPath(), testAutoApproved(), testManuallyDeclined)

So every second time 4 of 5 tests fails and every second time(+1) 1 of 5 test fails.
Do you have the same behavior?

Comment (dbudke) [reply src/test/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/ProcessScenarioTest.java]: Updated the readme to either user Spring Boot integration tests or the plain process tests.

**Pull request merged**